out/css/wirecard_wdoxidee_common.css

```diff
@@ -16,3 +16,16 @@
   border-color: #ebccd1;
   color: #c94442;
 }
+
+input[type='number'] {
```

Comment:
To avoid duplicating styles here, we might want to just use a `pattern` attribute on a regular `<input type="text">` field instead of using a `<input type="number">`.

Reply:
sounds good! @ps-mstifter please try that

Reply:
👍